pkgs/applications/networking/instant-messengers/slack/default.nix

```diff
@@ -73,7 +73,8 @@ in stdenv.mkDerivation {
 
     # Fix the symlink
     rm $out/bin/slack
-    ln -s $out/lib/slack/slack $out/bin/slack
+    makeWrapper $out/lib/slack/slack $out/bin/slack \
+      --set LD_LIBRARY_PATH ${rpath}
```

Comment:
I always thought, that LD_LIBRARY_PATH and `rpath` in the elf file have the same semantic. Maybe the rpath is stripped afterwards in the `fixupPhase`.